app/services/course/assessment/answer/auto_grading_service.rb

```diff
@@ -59,12 +63,13 @@ def grade(answer)
     answer
   end
 
-  # Evaluates and mark the answer as correct or not. This is supposed to be implemented by
-  # subclasses.
+  # This generic grader only marks the answer as correct and assigns the full grade.
+  # Actual autograding behaviour should be implemented by subclasses.
   #
   # @param [Course::Assessment::Answer] answer The answer to be evaluated.
   # @return [Integer] grade The grade of the answer.
   def evaluate(answer)
-    raise 'Not Implemented'
+    answer.correct = true
```

Comment:
Better use a new method to grade non-autograded answers instead. The result could become unexpected if an autograded question does not implement the method.